### PR TITLE
Greening: MASTER(Thunderjet)

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/integration/invoices/invoices.manuallyCreateLineFromPol.spec.js [hunk 1]
```diff
@@ -43,12 +58,16 @@ describe('ui-invoices: Invoice Line creation - based on POL', () => {
     invoiceLine.quantity = orderLine.cost.quantityPhysical;
     invoiceLine.subTotal = orderLine.cost.quantityPhysical * orderLine.cost.listUnitPrice;
   });
+
```
Comment (MaksimTerekhov) [inline cypress/integration/invoices/invoices.manuallyCreateLineFromPol.spec.js:61]: Trailing spaces not allowed

File: cypress/integration/orders/orders.createOrder.spec.js [hunk 2]
```diff
@@ -22,4 +30,4 @@ describe('orders: create an order', () => {
     Orders.createOrder(order);
     Orders.checkCreatedOrder(order);
   });
-});
+});
```
Comment (MaksimTerekhov) [inline cypress/integration/orders/orders.createOrder.spec.js:33]: Newline required at end of file but not found

File: cypress/integration/orders/orders.receivePieceFromOrder.spec.js [hunk 3]
```diff
@@ -11,25 +11,34 @@ import InteractorsTools from '../../support/utils/interactorsTools';
...
+        orderLine.physical.materialSupplier = response;
+        orderLine.eresource.accessProvider = response;
       });
     cy.getLocations({ query: `name="${OrdersHelper.mainLibraryLocation}"` })
       .then(location => { orderLine.locations[0].locationId = location.id; });
     cy.getMaterialTypes({ query: 'name="book"' })
       .then(materialType => { orderLine.physical.materialType = materialType.id; });
     cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
   });
+
```
Comment (MaksimTerekhov) [inline cypress/integration/orders/orders.receivePieceFromOrder.spec.js:36]: Trailing spaces not allowed

File: cypress/integration/orders/orders.unreceivePiece.spec.js [hunk 4]
```diff
@@ -57,4 +65,4 @@ describe('orders: Unreceive piece from Order', () => {
         InventoryInstance.checkHoldingsTable(OrdersHelper.mainLibraryLocation, 0, caption, barcode, 'On order');
       });
   });
-});
+});
```
Comment (MaksimTerekhov) [inline cypress/integration/orders/orders.unreceivePiece.spec.js:68]: Newline required at end of file but not found

**Review by MaksimTerekhov — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**